develop/api_guc.rst

```diff
@@ -472,3 +472,15 @@ citus.explain_all_tasks (boolean)
 ************************************************
 
 By default, Citus shows the output of a single, arbitrary task when running `EXPLAIN <http://www.postgresql.org/docs/current/static/sql-explain.html>`_ on a distributed query. In most cases, the explain output will be similar across tasks. Occasionally, some of the tasks will be planned differently or have much higher execution times. In those cases, it can be useful to enable this parameter, after which the EXPLAIN output will include all tasks. This may cause the EXPLAIN to take longer.
+
+citus.explain_analyze_sort_method (enum)
+************************************************
+
+Sorts the tasks in the output of EXPLAIN ANALYZE by execution time.
+The default value of citus.explain_analyze_sort_method is ``execution-time``. Also, running ``SET citus.explain_analyze_sort_method = 'taskId';`` will allow you to get the output sorted by task ids.
```

Comment:
We can remove the second sentence since you are also explaining it in the supported values